clients/drcachesim/tools/invariant_checker.cpp

```diff
@@ -669,6 +679,13 @@ invariant_checker_t::parallel_shard_memref(void *shard_data, const memref_t &mem
                       << "marker type " << memref.marker.marker_type << " value 0x"
                       << std::hex << memref.marker.marker_value << std::dec << "\n";
         }
+        // The interruption point is not a target for a return. Zero is push as
```

Comment:
Not sure the first sentence is needed?  I think it adds confusion?

Reply:
I've removed the first sentence.